File: official/docs/csharp/customsInfo/CreateCustomsInfo.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

CustomsInfo customsInfo = await CustomsInfo.Create(new Dictionary<string, object>() {
  { "customs_certify", true },
  {"customs_signer", "Steve Brule"},
  {"contents_type", "merchandise"},
  {"contents_explanation", ""},
  { "eel_pfc", "NOEEI 30.37(a)" },
  { "customs_items", new List<Dictionary<string, object>>()
      { new Dictionary<string, object>(){
        { "description", "T-shirt" },
        { "quantity", 1 },
        { "weight", 5 },
        {"value", 10},
        {"hs_tariff_number", "123456"},
        { "origin_country", "US" }
    } }
  },
  {"restriction_type", "none"},
  {"eel_pfc", "NOEEI 30.37(a)"}
});

Console.Write(customsInfo);

File: official/docs/java/customsInfo/CreateCustomsInfo.java
import com.easypost.EasyPost;

public class CreateCustomsInfo {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        Map<String, Object> customsItemMap = new HashMap<String, Object>();
        customsItemMap.put("description", "T-shirt");
        customsItemMap.put("quantity", 1);
        customsItemMap.put("value", 10);
        customsItemMap.put("weight", 5);
        customsItemMap.put("origin_country", "us");
        customsItemMap.put("hs_tariff_number", "123456");
        CustomsItem customsItem1 = CustomsItem.create(customsItemMap);

        List<CustomsItem> customsItemsList = new ArrayList<CustomsItem>();
        customsItemsList.add(customsItem1);

        Map<String, Object> customsInfoMap = new HashMap<String, Object>();
        customsInfoMap.put("customs_certify", true);
        customsInfoMap.put("customs_signer", "Steve Brule");
        customsInfoMap.put("contents_type", "merchandise");
        customsInfoMap.put("contents_explanation", "");
        customsInfoMap.put("eel_pfc", "NOEEI 30.37(a)");
        customsInfoMap.put("restriction_type", "none");
        customsInfoMap.put("customs_items", customsItemsList);

        CustomsInfo customsInfo = CustomsInfo.create(customsInfoMap);

        System.out.println(customsInfo);
    }
}
